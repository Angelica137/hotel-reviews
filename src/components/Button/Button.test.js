import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import Button from "./Button";

describe("the <Button /> component", () => {
  it("should render the correct children", () => {
    const children = "this is button";
    const component = shallow(<Button>{children}</Button>);

    expect(component.props().children).toEqual(children);
  });
});
