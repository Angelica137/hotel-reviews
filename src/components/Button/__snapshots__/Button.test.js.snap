// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the <Button /> component should render 1`] = `
<ForwardRef(styled.button)>
  This is a button
</ForwardRef(styled.button)>
`;
